src/Symfony/Component/HttpKernel/HttpKernelInterface.php

```diff
@@ -21,22 +21,27 @@
  */
 interface HttpKernelInterface
 {
-    public const MASTER_REQUEST = 1;
+    public const MAIN_REQUEST = 1;
     public const SUB_REQUEST = 2;
 
+    /**
+     * @deprecated Use {@see MAIN_REQUEST} instead
+     */
+    public const MASTER_REQUEST = self::MAIN_REQUEST;
```

Comment:
I cannot really trigger a deprecation for constants, can I? I could alternatively assign different values to `MASTER_REQUEST` and `MAIN_REQUEST`, but I fear that this might cause more trouble than it helps.

Reply:
I think this is fine.

Reply:
Could it be possible to use leverage PHP's lazy loading of consts? This would mean referencing internal class consts here, which would trigger the appropriate deprecation when loaded? I'm not sure it'll work, but let's give it a try?

Reply:
Yes, that seems to work.

```PHP
namespace Symfony\Component\HttpKernel;
use Symfony\Component\ErrorHandler\Debug;

require __DIR__.'/vendor/autoload.php';
Debug::enable();

echo HttpKernelInterface::MASTER_REQUEST.\PHP_EOL;
```
```
1
2021-03-22T12:21:55+00:00 [info] User Deprecated: Since symfony/http-kernel 5.3: "Symfony\Component\HttpKernel\HttpKernelInterface::MASTER_REQUEST" is deprecated, use MAIN_REQUEST instead.
```
```PHP
namespace Symfony\Component\HttpKernel;
use Symfony\Component\ErrorHandler\Debug;

require __DIR__.'/vendor/autoload.php';
Debug::enable();

echo HttpKernelInterface::MAIN_REQUEST.\PHP_EOL;
```
```
1
```